lpa-submissions-e2e-tests/cypress/integration/accuracyAppellantsSubmission.feature

```diff
@@ -0,0 +1,69 @@
+@wip
+Feature: Accuracy of the Apellants Submission
+
+  As a beta LPAI want to respond whether the information from the appellant accurately reflects the original planning application
+  so that the Planning Inspectorate can make a decision.
+
+  Scenario: AC01 Access 'Review the accuracy of the appellant's submission' sub-section
+    Given the user is on the Task List page
+    When the user selects the link 'Review the accuracy of the appellant's submission'
+    Then the user is presented with the page 'Review the accuracy of the appellant's submission'
+    And the Page Title is 'Review accuracy of appellant's submission - Appeal Questionnaire - Appeal a householder planning decision - GOV.UK'
+
+  Scenario: AC02 LPA User does not select Yes or No
+    Given the user is in the Review the accuracy of the appellant's submission sub-section
```

Comment:
Do we need to use the word sub-section or page? Effectively it is a different page.

Reply:
Sub-section is the wording used in the ticket so matching the ticket. This is how we write feature files is it not?

Reply:
As discussed with Pippa, we have agreed to rename these to pages for this and future tickets.